Game/Resources/Scripts/TemplateScript.cpp

```diff
@@ -0,0 +1,53 @@
+#include "TemplateScript.h"
+
+#include "Component/ComponentScript.h"
+#include "Component/ComponentTransform.h"
+
+#include "Main/Application.h"
+#include "Main/GameObject.h"
+#include "Module/ModuleInput.h"
+#include "Module/ModuleScene.h"
+
+#include "UI/Panel/InspectorSubpanel/PanelComponent.h"
+
+#include "imgui.h"
+
+
+
+TemplateScript* TemplateScriptDLL()
+{
+	TemplateScript *instance = new TemplateScript();
```

Comment:
Is there gonna be only 1 instance? Maybe then make it `static TemplateScript *instance = new TemplateScript();`, so you don't need to create a new object every single time you call `TemplateScriptDLL()`